lib/postcode_validation/use_case/address_list.rb

```diff
@@ -22,8 +22,8 @@ def execute(postcode:, country:)
 
       def result_payload(result, postcode)
         result.map do |address|
-          PostcodeValidation::Domain::Address.new(street_address: address['StreetAddress'],
-                                                  place: address['Place'])
+          PostcodeValidation::Domain::Address.new(street_address: address[:StreetAddress],
```

Comment:
This is how the PCApredict returns the results to us:
![screen shot 2017-04-26 at 10 12 25](https://cloud.githubusercontent.com/assets/8478978/25426970/e9a95794-2a68-11e7-9845-a0040ac45cc4.png)